test_simple_graph.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+import pytest
+# def test_display():
+#     """Test that the graph displays."""
+#     from simple_graph import 
+
+def test_add_node():
+    """Test if node added to graph."""
+    from simple_graph import Graph
+    graph = Graph()
+    graph.add_node('chicken')
+    assert 'chicken' in graph.container
+
+
+def test_not_in_graph():
+    """Test if the Node is in the Graph."""
+    from simple_graph import Graph
+    graph = Graph()
+    assert graph.has_node('chicken') is False
+
+
+def test_in_graph():
+    """Test if Node exists in Graph."""
+    from simple_graph import Graph
+    graph = Graph()
+    graph.add_node('monkey')
+    assert graph.has_node('monkey') is True
+
+
+def test_add_edge():
+    """Test if there is an edge between two nodes."""
+    from simple_graph import Graph
+    graph = Graph()
+    graph.add_node('chicken')
+    graph.add_node('egg')
+    graph.add_edge('egg', 'chicken')
+    assert 'chicken' in graph.container['egg']
+
```

Comment:
You need to also test that if one (or both) nodes aren't yet present in the graph, that g.add_edge() places them before drawing the edge.